protos/telemetry/telemetry.proto

```diff
@@ -20,6 +20,7 @@ service TelemetryService {
     rpc SubscribeFlightMode(SubscribeFlightModeRequest) returns(stream FlightModeResponse) {}
     rpc SubscribeHealth(SubscribeHealthRequest) returns(stream HealthResponse) {}
     rpc SubscribeRcStatus(SubscribeRcStatusRequest) returns(stream RcStatusResponse) {}
+    rpc SubscribeMavMessage(SubscribeMavMessageRequest) returns(stream MavMessageResponse) {}
```

Comment:
I wouldn't call it `MavMessage`, what about `StatusText`?

Reply:
Coll, will change the name. :)